base64/decode/decode.go

```diff
@@ -0,0 +1,38 @@
+package decode
```

Comment:
No strong opinion. Do not see any benefit to change it though. I just did it that way to have services implementing value modifiers in their own package. Then the struct got called `Service` because of copy paste from somewhere else.